systems/notification/api-gateway/pkg/rest/router_test.go

```diff
@@ -67,15 +67,14 @@ func TestPingRoute(t *testing.T) {
 func TestRouter_sendEmail(t *testing.T) {
```

Comment:
This test file is failing. Please, fix test case.

Reply:
Addressed